### ISSUE TITLE
crash Growatt plug-in

### ISSUE THREAD

**Issue by tonbor** [sincze/Domoticz-Growatt-Webserver-Plugin#10]
well I have 4 Domoticz at 4 pi's. One pi runs buster with latest stabel domoticz. I installed the growatt plug-in. Well it crashes every 3 or 4 days. The growatt script at the other pi is working correct for more then a year. Anybody else suffering from a instabel growatt plug-in?

**Comment by PvdGulik** [comment]
Yes, i reconize that behavior. Suddenly is stops getting data from Growatt. Restart Domoticz solves the problem for few days..
Using the plug-in in Domoticz 2020.2 on Buster on a Raspberry Pi 3B+

**Comment by tonbor** [comment]
Never solved, plugin crashes manager over abd over again. Reboot onlu sollution

**Comment by sincze** [comment]
This version as well ??

https://github.com/sincze/Domoticz-Growatt-Webserver-Plugin/blob/dac4b0e0de731ba7dbed0df1ba2adaeea73bbbe5/plugin.py
